README.md

```diff
@@ -39,13 +39,14 @@ $ tfplugindocs generate --help
 
 Usage: tfplugindocs generate [<args>]
 
-    --examples-dir <ARG>             examples directory                                                        (default: "examples")
+    --examples-dir <ARG>             examples directory based on provider-dir                                  (default: "examples")
     --ignore-deprecated <ARG>        don't generate documentation for deprecated resources and data-sources    (default: "false")
+    --provider-dir <ARG>             relative or absolute directory of provider code
```

Comment:
Not sure if it'll be a pain to have this in the actual help CLI, but can we add some mention that it will default to the working directory here as well?

Reply:
I believe there's at least two options that are available to denote this with the standard "(default: xxx)" output of the `--help` flag:

* Changing the default value of the flag from `""` (no default output) to `"."` and accounting for the new default in the later `os.Getwd()` logic. I'm guessing most folks running a CLI tool may be familiar with that directory syntax, but maybe not all.
* Changing the default value of the flag from `""` (no default output) to `os.Getwd()` and removing the later `os.Getwd()` logic. I'm guessing this could _potentially_ be lengthy or confusing as it changes each directory the command is run in.

We could also consider just putting words there outside the `(default: xxx)` output that says something along the lines of "relative or absolute directory of provider code _when running the command outside the root provider code directory_".

Do any of these seem preferable to you?

Reply:
(I'm going to implement the last option as part of this PR right now, but if changing the default flag value makes sense, can certainly do that instead.)

Reply:
We can change this up afterwards, if you have any further thoughts!

Reply:
Sorry about the delayed response! I'm cool with what you ended with. I think it'd be neat to display the working directory as the default but I can't remember a CLI tool I've used that has done that before 🤔.